lib/pengine/clone.c

```diff
@@ -811,7 +811,7 @@ pe__clone_xml(pcmk__output_t *out, va_list args)
         }
 
         out->message(out, crm_map_element_name(child_rsc->xml), show_opts,
-                     child_rsc, only_node, all);
+                     child_rsc, desc, only_node, all);
```

Comment:
To add an argument to an output message:
* Make sure every call to the message (in this case, all the resource output functions for primitive/group/clone/bundle) has the new argument in the same place
* Update `PCMK__OUTPUT_ARGS` for all of the message implementation functions to add the argument type in the correct place
* Add a `va_arg()` line in each message implementation function to grab the new argument into a variable

Then you can use the variable just as if it were a normal function argument.